experimental/sql/src/ibis_frontend.py

```diff
@@ -3,8 +3,6 @@
 # SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 
 from dataclasses import dataclass
-from modulefinder import Module
-from unittest import result
```

Comment:
I guess these are also unrelated changes? I'd move them to the same "clean-up" mini-PR I mentioned above.